dask/Dask_Hello_World.ipynb

```diff
@@ -0,0 +1,332 @@
+    "\n",
+    "To learn more about Dask, check out the documentation here: http://docs.dask.org/en/latest/\n",
+    "\n",
+    "Dask operates using a concept of a \"Client\" and \"workers\". The client tells the workers what tasks to perform and when to perform. Typically, we set the number of works to be equal to the number of computing resources we have available to us. For example, wer might set `n_workers = 8` if we have 8 CPU cores on our machine that can each operate in parallel. This allows us to take advantage of all of our computing resources and enjoy the most benefits from parallelization.\n",
+    "\n",
+    "Dask is a first class citizen in the world of General Purpose GPU computing and the RAPIDS ecosystem makes it very easy to use Dask with cuDF and XGBoost. As we see below, we can inititate a Cluster and Client using only few lines of code."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask_cuda import LocalCUDACluster\n",
+    "\n",
+    "cluster = LocalCUDACluster()\n",
+    "client = Client(cluster)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's show our current Dask status. We should see the IP Address for our Scheduler as well the the number of workers in our Cluster. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# show current Dask status\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also see the status and more information at the Dashboard, found at `http://<scheduler_uri>/status`. You can ignore this for now, we'll dive into this in subsequent tutorials."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Hello World\n",
+    "\n",
+    "Our Dask Client and Dask Workers have been setup. It's time to execute our first program in parallel. We'll define a function that takes some value `x` and adds 5 to it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def add_x_to_5(x):\n",
+    "    return x + 5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll iterate through our `n_workers` and create an execution graph, where each worker is responsible for taking it's ID and passing it to the function `add_x_to_5`. For example, Dask Worker 2 will result in the value 7.\n",
+    "\n",
+    "An important thing to note is that the Dask Workers aren't actually executing these results - we're just defining the execution graph for our Dask Client to execute later. The `delayed` function wrapper ensures that this computation is in fact \"delayed\" and not executed on the spot."
```

Comment:
What's the motivation for introducing `delayed` here? Why not dask_cudf.read_csv and some sums or counts on the result df? You can still explain lazy execution with `.compute()` and `.wait()`

I think `delayed` makes sense in the context of doing something entirely worker-local. We used them in the mortgage workflow partly because acquisition and performance data was completely independent across quarters.. e.g. we didn't need to use cuDF operations that ran across file boundaries. We also did it that way initially because dask_cudf functions weren't ready.

Do you think it makes sense to move an explanation of `delayed` to the bottom and explain how you can do worker-local only operations like apply_rows, or something else not yet exposed in dask_cudf? An example of `delayed` with `dask_cudf.from_delayed` would be nice.

Reply:
This review went in the wrong place. Moving..